.github/workflows/build.yml

```diff
@@ -5,63 +5,74 @@ on:
       - main
 
 jobs:
-  build:
+  get-version:
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      id-token: write
+    outputs:
+      version: ${{ steps.get_version.outputs.version }}
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Install Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
       - name: Get version
         id: get_version
         run: echo "version=$(grep "version" Cargo.toml | head -n1 | cut -d '"' -f2)" >> $GITHUB_OUTPUT
       - name: Compare version to NPM
         run: |
-          NPM_VERSION=$(npm view create-o7-app version)
+          NPM_VERSION=$(curl -s https://registry.npmjs.org/create-o7-app | jq -r '.["dist-tags"].latest')
           MY_VERSION=${{ steps.get_version.outputs.version }}
           if [ "$NPM_VERSION" == "$MY_VERSION" ]; then
             echo "NPM version is the same as the version in Cargo.toml, exiting"
             exit 1
```

Comment:
Can you look into https://stackoverflow.com/a/61832535 to conditionally skip the other tasks instead of failing the whole workflow?

Reply:
I added the condition